blockchain/stakeversion.go

```diff
@@ -18,6 +19,13 @@ var (
 		"not found")
 )
 
+func stakeMajorityCacheVersionKey(version uint32, hash *chainhash.Hash) [stakeMajorityCacheKeySize]byte {
```

Comment:
Comment with intended use please.  How is it intended to be used?  Is it only for a single cache, or is it intended to be used in multiple?